content/en/tracing/troubleshooting/agent_apm_metrics.md

```diff
@@ -214,14 +210,6 @@ Number of events processed.
 : **Type**: Gauge<br>
 Time it took to flush a payload to the Datadog API.
 
-`datadog.trace_agent.trace_writer.payloads`
-: **Type**: Count<br>
-Number of payloads processed.
-
-`datadog.trace_agent.trace_writer.payloads`
-: **Type**: Count<br>
-Number of payloads sent.
-
```

Comment:
I wouldn't remove it altogether, it's probably better to remove one of them and make sure the description of the one that's left remains correct.